diff --git a/node_modules/grunt-mocha-istanbul/tasks/index.js b/node_modules/grunt-mocha-istanbul/tasks/index.js
index 709b87a..c607749 100644
--- a/node_modules/grunt-mocha-istanbul/tasks/index.js
+++ b/node_modules/grunt-mocha-istanbul/tasks/index.js
@@ -5,7 +5,7 @@ module.exports = function (grunt) {
 
     function getMochaPath() {
         try {
-            return require.resolve('mocha/bin/_mocha');
+            return require.resolve('ts-mocha/bin/ts-mocha');
         } catch (ignored) {
             grunt.fail.fatal('Mocha peer dependency missing.  Please "npm install mocha"');
         }
